template/accordion/accordion-group.html

```diff
@@ -1,4 +1,4 @@
-<div class="panel panel-default">
+<div class="panel panel-{{panelClass || 'default'}}">
```

Comment:
I think I would prefer this be `{{panelClass || 'panel-default'}}` - gives the users of the library a little more flexibility without any cost.